.rubocop.yml

```diff
@@ -1,4 +1,7 @@
+inherit_from: .rubocop_todo.yml
+
```

Comment:
Rubocop, as the linter, should be treated as blocking the build. I don't want to maintain a todo list like we have in Hashie.

Honestly, I would rather use `rubyfmt` when it's ready and instead use a subset of Rubocop.

Reply:
Removed the TODO.